lib/vscode/src/vs/server/node/channel.ts

```diff
@@ -350,6 +350,13 @@ class VariableResolverService extends AbstractVariableResolverService {
 			getConfigurationValue: (_: URI, section: string): string | undefined => {
 				return args.resolvedVariables[`config:${section}`];
 			},
+			getAppRoot: (): string | undefined => {
```

Comment:
Let's link `../../workbench/api/common/extHostDebugService.ts` above as a reference for the `VariableResolverService` class.

With that in mind, looks like it should be similar to `getExecPath`. 

Something like:
```typescript
 return (args.resolverEnv && args.resolverEnv['VSCODE_CWD']) || env['VSCODE_CWD'] || process.cwd();
```

Reply:
Ah gotcha! Okay I tried that and I don't see any errors lol